cli/src/main/kotlin/Main.kt

```diff
@@ -141,4 +165,25 @@ object Main : CommandWithHelp() {
 
         println()
     }
+
+    private fun loadConfig(): OrtConfiguration {
+        val argConfig = ConfigFactory.parseMap(configArguments, "Command line").withOnlyPath("ort")
+        val systemConfig = ConfigFactory.systemProperties().withOnlyPath("ort")
```

Comment:
I'm wonder whether *if* we consider JVM system properties at all (I'd be also fine with dropping them; what's the concrete use-case?), should we make them a higher priority than command line arguments?

Also, I'd slightly prefer the name `propertiesConfig`, as `systemConfig` sounds to me like a system-wide (i.e. not user-specific) configuration file.

Reply:
The use case would be to have a convenient way to override configuration when not using CLI, but that would require architectural changes to where we load the configuration, so I have removed it for now.